README.md

```diff
@@ -71,12 +71,12 @@ On occasion certficates can fail to get updated and it is nice to know before it
 ### Bonus Commands
 
 If you are fault finding on a swarm you will find yourself looking up ports
-from @signs and @signs to ports... so
+from Atsigns and Atsigns to ports... so
```

Comment:
Change for readability to `If you are fault finding on a swarm you will find yourself looking up ports
from Atsigns and looking up Atsigns from ports... so`